src/OpenLoco/OpenLoco.cpp

```diff
@@ -336,6 +338,23 @@ namespace OpenLoco
         call(0x004BE5EB, regs);
     }
 
+    // 0x004BE65E
+    void exitCleanly()
```

Comment:
```suggestion
    [[ noreturn ]] void exitCleanly()
```

Reply:
Your call to exit will never return. It can potentially cause compiler warnings if you rely on this.

Reply:
Right, we indeed had to deal with MSVC warning C4702. Your solution is better than disabling the warning — let's go with that.

Reply:
Turns out it's still triggering the C4702 warning, even with the noreturn attribute. I will reinstate the pragma line suppressing it, then.

Reply:
The reason for noreturn would be something lets say something like this:
```
[[noreturn]] void exitCleanly();

int foo()
{
   if(_requestExit)
   {
       exitCleanly();
   }
   else
   {
       return 1;
   }
}
```
This would warn that not all paths return a value if exitCleanly would not have the ```[[noreturn]]``` attribute